internal/cmd/error.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import "fmt"
+
+// CLI Errors are user facing errors that is formatted.
+// Should be used for communication, rather than a stacktrace.
+type Error struct {
+	// Error that is caused by the system, used for debugging
+	// Only set this if you need it to be printed as part of the messsage
+	OriginalError error
+
+	// Human readable error message that user will read on the CLI.
+	// These should be full sentences, rather than a stack trace.
+	// May include suggestions.
+	Message string
+}
+
+func (e Error) Error() string {
+	if e.OriginalError != nil {
+		if len(e.Message) == 0 {
+			return fmt.Sprintf("Internal error:\n%v", e.OriginalError)
+		}
+
+		// Strip '.' at the end when message includes the original error
+		if string(e.Message[len(e.Message)-1]) == "." {
+			e.Message = e.Message[:len(e.Message)-1]
+		}
+		return fmt.Sprintf("%v:\n       %v", e.Message, e.OriginalError)
```

Comment:
Should this indent be a `\t`?

Reply:
\t seems to be off by one space: 
<img width="91" alt="image" src="https://user-images.githubusercontent.com/20054646/165995166-2d8f07a5-d944-47ea-a620-68dd22d123b1.png">